src/core/loadbytecode.c

```diff
@@ -7,6 +7,43 @@ static void run_load(MVMThreadContext *tc, void *sr_data);
 static void mark_sr_data(MVMThreadContext *tc, MVMFrame *frame, MVMGCWorklist *worklist) {
     MVM_gc_worklist_add(tc, worklist, &frame->special_return_data);
 }
+void MVM_run_comp_unit(MVMThreadContext *tc, MVMCompUnit *cu) {
```

Comment:
If this is just for internal use (within this file), it should be marked `static`, and the convention is such things don't get an `MVM_` prefix.